kuksa_databroker/databroker-cli/src/help.rs

```diff
@@ -0,0 +1,1029 @@
+/********************************************************************************
+* Copyright (c) 2023 Contributors to the Eclipse Foundation
+*
+* See the NOTICE file(s) distributed with this work for additional
+* information regarding copyright ownership.
+*
+* This program and the accompanying materials are made available under the
+* terms of the Apache License 2.0 which is available at
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* SPDX-License-Identifier: Apache-2.0
+********************************************************************************/
+
+fn set_connected_prompt(interface: &Arc<Interface<DefaultTerminal>>, text: str) {
+    let mut _text;
+    _text = text;
+    let connected_prompt = format!(
+        "\x01{prefix}\x02{text}\x01{suffix}\x02 > ",
+        prefix = Color::Green.prefix(),
+        text = _text,
+        suffix = Color::Green.suffix()
+    );
+    interface.set_prompt(&connected_prompt).unwrap();
+}
+
+fn set_disconnected_prompt(interface: &Arc<Interface<DefaultTerminal>>) {
+    let disconnected_prompt = format!(
+        "\x01{prefix}\x02{text}\x01{suffix}\x02 > ",
+        prefix = Color::Red.prefix(),
+        text = "not connected",
+        suffix = Color::Red.suffix()
+    );
+    interface.set_prompt(&disconnected_prompt).unwrap();
+}
+
+fn to_uri(uri: impl AsRef<str>) -> Result<Uri, String> {
+    let uri = uri
+        .as_ref()
+        .parse::<tonic::transport::Uri>()
+        .map_err(|err| format!("{err}"))?;
+    let mut parts = uri.into_parts();
+
+    if parts.scheme.is_none() {
+        parts.scheme = Some("http".parse().expect("http should be valid scheme"));
+    }
+
+    match &parts.authority {
+        Some(_authority) => {
+            // match (authority.port_u16(), port) {
+            //     (Some(uri_port), Some(port)) => {
+            //         if uri_port != port {
+            //             parts.authority = format!("{}:{}", authority.host(), port)
+            //                 .parse::<Authority>()
+            //                 .map_err(|err| format!("{}", err))
+            //                 .ok();
+            //         }
+            //     }
+            //     (_, _) => {}
+            // }
+        }
+        None => return Err("No server uri specified".to_owned()),
+    }
+    parts.path_and_query = Some("".parse().expect("uri path should be empty string"));
+    tonic::transport::Uri::from_parts(parts).map_err(|err| format!("{err}"))
+}
+
+fn print_logo(version: impl fmt::Display) {
```

Comment:
Is this identical to "print_logo" in cli.rs?

Reply:
ah yes :) forgot to remove